packages/angular_devkit/build_angular/package.json

```diff
@@ -101,7 +101,7 @@
     "zone.js": "^0.10.0"
   },
   "peerDependencies": {
-    "@angular/compiler-cli": ">=9.0.0 < 10",
+    "@angular/compiler-cli": ">=9.1.0 < 10",
```

Comment:
Does this mean that the 9.0.x CLI packages we have are incompatible with FW 9.1.x?

What happens when a user updates CLI but not FW, and vice versa?

Reply:
- CLI 9.0 is compatible with FW 9.1.
- FW 9.1 is compatible with CLI 9.0
- CLI 9.1 is NOT compatible with FW 9.0 (That's because this API/CLI options) are only available in 9.1

Reply:
Can we then detect on the CLI side if we should use the new options or not? If we can't use the new options, then we should continue using the 9.0 ones.

The alternative would be to error out, but given our support policy I don't think that's a real option.

Reply:
If you kept the `pathMappings` stuff in there, then it would be compatible I think, at least for the sync integration. For the async integration, ngcc might complain about an unknown `--tsconfig` option.

Reply:
Umm, the strict unknown option handling was added in 9.1, so we should be fine.

Reply:
LGTM, thanks for these changes!